docs/GETTING-STARTED.md

```diff
@@ -68,6 +75,74 @@ app.post('/login', function(req, res) {
 });
 ```
 
+As said earlier, API documentation filters could be put in place before having such API rendered on the JSON file. A sample is shown in [app.js](../example/v2/app.js) where some form of filtering is done.
+```javascript
+function jsDocFilter(jsDocComment) {
+    // Do filtering logic here in order to determine whether
+    // the JSDoc under scrunity will be displayed or not.
+    // This function must return boolean. `true` to display, `false` to hide.
+    const docDescription = jsDocComment.description;
+
+    const features = docDescription.indexOf('feature') > -1;
```

Comment:
I like the approach of storing checks in variables. For readability, you can name them `hasFeatureX`, `hasFeatureY`, etc. Also, since we are having modern JS version, you can freely use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes